README.md

```diff
@@ -0,0 +1,179 @@
+
+An Awesome books store project, where you can enter the name and author of the books and have it stored, Developed using Html, Css and Javascript
+
+## 🛠 Built With <a name="built-with"></a>
+
+1-HTML
+2-CSS
+3-JS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add index.html file.
+- Add style.css file.
+- Add app.js file.
+- Add Linters.
+- Add Header to the index.html file.
+- Add global styling variables to style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+1-Web Browser
+2-Code Editor
+3-Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this Command:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Errytagedesign/microverse-portfolio.git
+```
+
+### Install
+
+First you need to install the node package files
+Run this command:
+
+```sh
+  cd microverse-portfolio
+  npm install
+```
+
+### Usage
+
+- In order to use this project you either create branch to add more features.
+- But in order to run the project open index.html with live server
+
+### Run Linters
+
+I have added the linters or tests in order to avoid anykind of typos and ensure the good practices in this project. So while using this repository if you need to check your work in html and css you can use below commands
+
+- in order to run the linter test on index.html:
+
+```sh
+ npx hint .
+```
+
+- in order to run the linter test on stylesheets:
+
+```sh
+ npx styleint "**/*.{css,scss}"
+```
+
+### Deployment
+
+- You can easily use github pages, netlify or vercel to deploy your project
+  <a href="https://docs.github.com/en/pages/quickstart">Useful link about Github Pages</a>
+  <a href="https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/">Step by Step guide to deploy using netlify</a>
+  <a href="https://www.programonaut.com/host-your-application-for-free-with-vercel-step-by-step/">Step by Step guide to deploy using vercel</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Sodeeq Awoyemi
+
+- GitHub: [@githubhandle](https://github.com/Errytagedesign)
+- Twitter: [@twitterhandle](https://twitter.com/errytage)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/errytagedesigns/)
+
+👤 Abu Talha
+
+- GitHub: [@githubhandle](https://github.com/AbuTalha3)
+- Twitter: [@twitterhandle](https://twitter.com/errytage)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/errytagedesigns/)
```

Comment:
- [ ] These links lead to the wrong person. They are supposed to lead to AbuTahlha3's social sites. Please have them fix or deleted if AbhuTalha does not have these social accounts. I would suggest that he creates them. 👍🏾